File: 9.palindrome-number.cpp
// @leet start
#import <string>
using namespace std;
class Solution {
public:
  bool isPalindrome(int x) {
		bool ans=true;
    string string1 = to_string(x);
		int left=0;
		int right=string1.size()-1;
		while (left<right) {
				if (string1[left]!=string1[right]) {
					ans=false;
					break;
				}	
			left++;
			right--;
		}
    return ans;
  }
};
// @leet end

File: 9.palindrome-number.js
// @leet start
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
	var string = x.toString();
	var left = 0;
	var right = string.length-1;
	while (left < right) {
		if (string[left]  !== string[right]) {
		return false	
		}
		left++;
		right--;
	}
	return true
};
// @leet end
